agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -808,7 +808,8 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
         LOG(DEBUG) << "CLIENT_BSS_STEER (802.11v) for sta_mac = " << sta_mac
                    << " to bssid = " << target_bssid
                    << " channel = " << int(request->params().target.channel);
-        ap_wlan_hal->sta_bss_steer(sta_mac, target_bssid, request->params().target.channel,
+        ap_wlan_hal->sta_bss_steer(sta_mac, target_bssid, request->params().target.operating_class,
```

Comment:
Why can't this be squashed to the previous commit?

Reply:
sure , just after the 20 commits in my last pr i felt i had to split something :)